docs/release-checklist.md

```diff
@@ -8,20 +8,21 @@ Martha Release Checklist
 
 ## Steps
 
-1. Create a git tag for the new version of Martha to be deployed and push it to the github repo (if done locally).
+- [ ] Create a Github ticket for Martha release and copy this checklist there.
+- [ ] Create a git tag for the new version of Martha to be deployed and push it to the github repo (if done locally).
     - e.g.: `git tag -a 1.5.2 -m "1.5.2"`
     - [tags for past releases](https://github.com/broadinstitute/martha/releases) to figure out what the last tag was
     - [compare the latest code with the last version](https://github.com/broadinstitute/martha/compare) to figure out what the next tag should be using [semantic versioning](https://semver.org/)
-    - [if you're looking for even more info on tags](https://git-scm.com/book/en/v2/Git-Basics-Tagging#_creating_tags) 
-1. This should trigger a Circle CI build of a docker image with that new tag as its name. Confirm that this has happened
+    - [if you're looking for even more info on tags](https://git-scm.com/book/en/v2/Git-Basics-Tagging#_creating_tags)
+- [ ] This should trigger a Circle CI build of a docker image with that new tag as its name. Confirm that this has happened
    by listing the google tags.
     - Run: `gcloud container images describe us.gcr.io/broad-dsp-gcr-public/martha:<tag>`
     - e.g.: `gcloud container images describe us.gcr.io/broad-dsp-gcr-public/martha:1.5.2`
-1. Deploy and test on dev (including manual test)
+- [ ] Deploy and test on dev (including manual test)
     1. Go to the [Jenkins manual deploy project](https://fc-jenkins.dsp-techops.broadinstitute.org/job/martha-manual-deploy/) and click on the "Build with Parameters" link.  For the parameters, you want to select the tag you just created and "dev" as the TARGET.  Do not proceed until the job ends successfully.
         - You will need to be on the Broad network in order to have access.
     1. [Run the full manual test on the dev environment](https://docs.google.com/document/d/1-SXw-tgt1tb3FEuNCGHWIZJ304POmfz5ragpphlq2Ng)
-1. Deploy and test on the other non-prod environments
+- [ ] Deploy and test on the other non-prod environments
     1. Go to the [Jenkins manual deploy project](https://fc-jenkins.dsp-techops.broadinstitute.org/job/martha-manual-deploy/) and click on the "Build with Parameters" link.  For the parameters, you want to select the tag you just created and "staging" as the TARGET.  Do not proceed until the job ends successfully.
```

Comment:
ToL: While you're here, via the [Release Testing Guidelines](https://docs.google.com/document/d/1Ej50XUEPwLQtjK6mQF9-xx7DuXyIhEs3UP8aii0bpsk/edit) one should be releasing to alpha _before_ staging, not after.

But that can be a separate ticket.

![release environments](https://user-images.githubusercontent.com/791985/110666487-60dbe280-8197-11eb-96b6-adc54d0f620e.png)

Reply:
I've made an update in this PR